Classes/Service/AuthenticationService.php

```diff
@@ -315,18 +322,10 @@ protected function convertResourceOwner(array $info)
         $queryBuilder->getRestrictions()->removeAll();
         $row = $queryBuilder
             ->select('*')
-            ->from($userTable)
-            ->where(
-                $queryBuilder->expr()->in('pid', $queryBuilder->createNamedParameter(
-                    GeneralUtility::intExplode(',', $this->config['usersStoragePid']),
-                    Connection::PARAM_INT_ARRAY
-                )),
-                $queryBuilder->expr()->orX(
-                    $queryBuilder->expr()->eq('tx_oidc', $queryBuilder->createNamedParameter($info['sub'])),
-                    $queryBuilder->expr()->eq('username', $queryBuilder->createNamedParameter($info['email']))
-                )
-            )
-            ->execute()
+            ->from($userTable)->where($queryBuilder->expr()->in('pid', $queryBuilder->createNamedParameter(
+            GeneralUtility::intExplode(',', $this->config['usersStoragePid']),
+            Connection::PARAM_INT_ARRAY
+        )), $queryBuilder->expr()->or($queryBuilder->expr()->eq('tx_oidc', $queryBuilder->createNamedParameter($info['sub'])), $queryBuilder->expr()->eq('username', $queryBuilder->createNamedParameter($info['email']))))->executeQuery()
```

Comment:
Any reason this is all stuffed into a single line? I consider the former formatting way more readable.

Reply:
But i can change that, Depends in codestyle. Do you have one?

Reply:
Okay, Rector is not famous for code-style ;-)
PSR-12 is kind of the minimum, I would say. But I *think* Xavier is more inclined to sticking with TYPO3's (even stricter) style.
I don't want to nitpick around with code-style. We don't have PHPCS in place (yet), so better get things working and cleanup the style later.

Reply:
I don't care at all about Rector :) That's totally unreadable, and clearly something I don't want to wait "for some future change" to get fixed.